src/utils/make_config_hex.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+INPUT_FILE=$1
+DESTINATION_DIR=$2
+
+TARGET_ADDRESS=0x080FC000
+
+srec_cat ${INPUT_FILE} -binary -offset ${TARGET_ADDRESS} \
```

Comment:
is srec_cat available on all build environments and the same version?
(osx/linux/cygwin)

The latest cygwin installer doesn't have a srecord package.
build instructions are here:
http://srecord.sourceforge.net/windows.html

if we settle on using srec then it should be added as a downloadable or buildable tool to the project like we do for dfuutil, openocd, stmflasher, gcc, etc.

Reply:
No, this is just a shim that I built to make it possible to use this (in my environment) in a consistent way.
Contributions of shims for other platforms are welcome.

Reply:
Understood.

I feel that since this work is a major part of the whole point of the unified targets / 4.1 focus/release it makes a lot of sense to me to apply some polish to this part of the build process.